dev/app.js

```diff
@@ -6,7 +6,6 @@ var search = instantsearch({
   apiKey: '6be0576ff61c053d5f9a3225e2a90f76',
   indexName: 'instant_search',
   urlSync: {
-    useHash: true
```

Comment:
Yep makes sense! Will make the modification to be sure that true is the value we want :)